changelog.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Focus movement on all diagrams.
 * Index page:
   * Add a search bar to filter diagrams.
+  * Add drawer to select template of a diagram.
```

Comment:
Suggestion : `Add drawer to create diagram from a template.`

Reply:
done